meetuppizza/tests.py

```diff
@@ -36,9 +36,13 @@ def test_user_is_created(self):
     self.assertFalse(user == None)
 
   def test_super_user_is_created(self):
```

Comment:
Don't be afraid of more descriptive names here. Think through the what, when, why, where, and how of what you're testing. A better name for this might be something like `test_user_migrations_create_default_super_user` or `test_default_super_user_created_by_user_migrations`.